packages/input-range/src/LionInputRange.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { LocalizeMixin, formatNumber } from '@lion/localize';
+import { FieldCustomMixin } from '@lion/field';
+import { LionInput } from '@lion/input';
+import { MinMaxNumber } from '@lion/validate/index.js';
+import { html } from '@lion/core';
+
+/**
+ * LionInputRange: extension of lion-input
+ *
+ * @customElement `lion-input-range`
+ * @extends LionInput
+ */
+export class LionInputRange extends FieldCustomMixin(LocalizeMixin(LionInput)) {
+  static get properties() {
+    return {
+      min: Number,
+      max: Number,
+      unit: String,
+      step: {
+        type: Number,
+        reflect: true,
+      },
+      ticks: {
+        type: Array,
+      },
+    };
+  }
+
+  connectedCallback() {
+    if (super.connectedCallback) super.connectedCallback();
+    this.type = 'range';
+  }
+
+  constructor() {
+    super();
+    this.parser = value => parseFloat(value);
+    if (this.min && this.max) {
+      this.defaultValidators.push(new MinMaxNumber({ min: this.min, max: this.max }));
```

Comment:
It is only possible to get wrong values when the modelValue is set programmatically. Don't think this is needed.